Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM  node:alpine
+
+# needed to build (python needed for compiling node-sass, for instance...)
+RUN apk add --no-cache \
```

Comment:
it seems strange that you need so much dependencies… especially things like openssh, git, vim, bash, etc…